internal/cmd/run.go

```diff
@@ -213,11 +202,9 @@ func getRunCmd() *cobra.Command {
 	cmd.Flags().BoolVar(&runStdinFlag, "stdin", false, "Take input from stdin (same format as the --raw option)")
 
 	// Feature flag
-
-	cmd.Flags().BoolVar(&overdraftFeatureFlag, flags.ExperimentalOverdraftFunctionFeatureFlag, false, "enables the experimental overdraft() function")
-	cmd.Flags().BoolVar(&oneOfFeatureFlag, flags.ExperimentalOneofFeatureFlag, false, "enable the experimental oneof combinator")
-	cmd.Flags().BoolVar(&accountInterpolationFlag, flags.ExperimentalAccountInterpolationFlag, false, "enables an account interpolation syntax, e.g. @users:$id:pending")
-	cmd.Flags().BoolVar(&midScriptFunctionCallFeatureFlag, flags.ExperimentalMidScriptFunctionCall, false, "allows to use function call as expression, and to use any expression when definining variables")
+	cmd.Flags().StringSliceVar(&flags_, "flags", nil, fmt.Sprintf("the feature flags to pass to the interpreter. Currently available flags: %s",
```

Comment:
Very bad global vars.
If you try to test while parallelizing, it will probably break.